src/main/kotlin/no/nav/syfo/personstatus/application/oppfolgingsoppgave/OppfolgingsoppgaveDTO.kt

```diff
@@ -19,4 +20,43 @@ data class OppfolgingsoppgaveDTO(
     val tekst: String?,
     val oppfolgingsgrunn: String,
     val frist: LocalDate?,
+){
+    companion object {
+        fun fromOppfolgingsoppgaveNewDTO(oppfolgingsoppgaveNewDTO: OppfolgingsoppgaveNewDTO): OppfolgingsoppgaveDTO =
+            OppfolgingsoppgaveDTO(
+                uuid = oppfolgingsoppgaveNewDTO.uuid,
+                updatedAt = oppfolgingsoppgaveNewDTO.updatedAt,
+                createdAt = oppfolgingsoppgaveNewDTO.createdAt,
+                createdBy = oppfolgingsoppgaveNewDTO.sisteVersjon().createdBy,
+                tekst = oppfolgingsoppgaveNewDTO.sisteVersjon().tekst,
+                oppfolgingsgrunn = oppfolgingsoppgaveNewDTO.sisteVersjon().oppfolgingsgrunn,
+                frist = oppfolgingsoppgaveNewDTO.sisteVersjon().frist
+            )
+    }
+}
+
+data class OppfolgingsoppgaverNewResponseDTO(
+    val oppfolgingsoppgaver: Map<String, OppfolgingsoppgaveNewDTO>
 )
+
+data class OppfolgingsoppgaveNewDTO(
```

Comment:
Vil gjerne ha forslag på annet navn ettersom det er tiltenkt at dette er en type som skal leve ved siden av eksisterende: `OppfolgingsoppgaveDTO`. Så enten kan man gi den eksisterende et nytt navn eller så bør nye hete noe annet.

Reply:
Kan vel endre tilbake til det "gamle" navnet til slutt (og merge PR'ene omtrent samtidig på ettermiddagen)?

Reply:
Ah, og her skal gammel og ny versjon fortsette å finnes: er forskjellen at den nye har med info om alle versjoner? I så fall kunne man finne et navn som reflekterer dette?

Reply:
Begge typene må selvfølgelig ikke eksistere samtidig, men jeg tenkte det var et naturlig sted å stoppe eller så må man vurdere å refaktore `syfooversiktsrv` og det tror jeg ikke er så hensiktsmessig eller at vi tjener noe særlig på. 

Og ja det stemmer. Forskjellen er nå at den kommer med alle versjonene hvor den nyeste/siste ligger først i listen. Skal tenke litt mer på navn eller så synes jeg forslaget "WithVersions" var relativt beskrivende.

Kunne også vurdert å kalle den eksisterende `aktivOppfolgingsoppgaveDTO` (eller `active` om ønskelig) og at den nye tar det eksisterende navnet: `OppfolgingsoppgaveDTO`

Reply:
Ser nå at det å kalle det noe med `aktiv` blir i grunn litt feil ettersom alle disse oppgavene er aktive for alle personene. Så jeg tror kanskje jeg ville gitt den eksisterende navnet: `OppfolgingsoppgaveLatestVersionDTO` eller muligens `newest`. 

Jeg tenker det kan gi mer mening sånn at man overordnet kan tenke på at `Oppfolgingsoppgave` er et begrep som alltid inneholder alle versjonene.

Reply:
Tror "LatestVersion" er mer i sync med bergrepsbruken vi har ellers (dvs bedre enn enn "NewestVersion").